packages/vite/src/node/server/index.ts

```diff
@@ -402,6 +408,9 @@ export async function _createServer(
     ws,
     moduleGraph,
     resolvedUrls: null, // will be set on listen
+    _setInternalServer(_server: ViteDevServer) {
+      server = _server
+    },
```

Comment:
I think it would be nicer to put this at the bottom with the other internal functions for consistency.

Reply:
Done 👍🏼